<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flame spectrometer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="PlotWidget" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="ParamBox">
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>170</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Parameters</string>
        </property>
        <widget class="QPushButton" name="runButt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>27</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Exposure</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>No.Accum</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="expEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="noAccEdit">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>60</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="setParmsButt">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>87</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="elDarkChk">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>120</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>El. dark correction</string>
         </property>
        </widget>
        <widget class="QPushButton" name="locButt">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>40</y>
           <width>88</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
        <widget class="QLabel" name="locLabel">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>37</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>./</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="nameEdit">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>70</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButt">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>70</y>
           <width>88</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>100</y>
           <width>58</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="formatCombo">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>100</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>ASCII XY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ASCII X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>float x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>double x</string>
          </property>
         </item>
        </widget>
        <widget class="QGroupBox" name="pwrRadioBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>111</width>
           <height>101</height>
          </rect>
         </property>
         <property name="title">
          <string>Take as</string>
         </property>
         <widget class="QRadioButton" name="backChk">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>101</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Background</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="sigChk">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>101</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Signal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="refChk">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>101</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Reference</string>
          </property>
         </widget>
        </widget>
        <widget class="QCheckBox" name="absChk">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>140</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Absorption</string>
         </property>
        </widget>
        <widget class="QLabel" name="expLabel">
         <property name="geometry">
          <rect>
           <x>246</x>
           <y>33</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
        <widget class="QLabel" name="noAccLabel">
         <property name="geometry">
          <rect>
           <x>248</x>
           <y>64</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>expEdit</tabstop>
  <tabstop>runButt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
